v1.2/alter-sequence.md

```diff
@@ -34,11 +36,50 @@ table td:first-child {
 
  Parameter | Description
 -----------|------------
+`IF EXISTS` | Modify the sequence only if it exists; if it does not exist, do not return an error.
+`name` | The name of the sequence you want to modify.
+`INCREMENT` | The new value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.
+`MINVALUE` | The new minimum value of the sequence.
+`MAXVALUE` | The new maximum value of the sequence.
+`START` | The new starting value of the sequence.
```

Comment:
Doesn't work yet

Reply:
This doesn't set the sequence value, but it sets:

- the value that it goes to if you do `ALTER SEQUENCE foo RESTART` (not implemented yet)
- the value that it will wrap around to if you hit the `MAXVALUE` and `CYCLE` is on. This should work. E.g.:

```
CREATE SEQUENCE foo MAXVALUE 5 CYCLE
=> CREATE SEQUENCE
... increment up to 5 ...
ALTER SEQUENCE foo START -1000
=> ALTER SEQUENCE
SELECT nextval('foo')
=> 5
```

If we hadn't done the alter sequence, it would have cycled back to 1 (the default start)